Documentation/cmdref/kvstore.rst

```diff
@@ -50,13 +50,15 @@ etcd
 When using etcd, one of the following options need to be provided to configure the
 etcd endpoints:
 
-+---------------------+---------+---------------------------------------------------+
-| Option              |  Type   | Description                                       |
-+---------------------+---------+---------------------------------------------------+
-| etcd.address        | Address | Address of etcd endpoint                          |
-+---------------------+---------+---------------------------------------------------+
-| etcd.config         | Path    | Path to an etcd configuration file.               |
-+---------------------+---------+---------------------------------------------------+
++------------------------------------+---------+------------------------------------------------------+
+| Option                             |  Type   | Description                                          |
++------------------------------------+---------+------------------------------------------------------+
+| etcd.address                       | Address | Address of etcd endpoint                             |
++------------------------------------+---------+------------------------------------------------------+
+| etcd.config                        | Path    | Path to an etcd configuration file.                  |
++------------------------------------+---------+------------------------------------------------------+
+| etcd.initconnectiontimeout         | Time    | Initial connection etcd timeout(Default: 15minutes ).|
```

Comment:
This might not be the best place to add this documentation. The line above this section is `one of the following options need to be provided to configure the etcd endpoints:` and this new option isn't quite the same thing (it isn't required, for example, and setting doesn't replace the need for one of the other two (I think?). Looking through the doc, there isn't really a good place to put it, so it might be best to create a new section under the etcd section.

Is this timeout the time until cilium returns health-bad when it cannot connect to etcd? or is it an actual single-connection timeout? It might be good to explain the effect of this value a little more.